### ISSUE TITLE
Amazon listing date parsing

### ISSUE THREAD

**Issue by iamsr** [amz-tools/amazon-sp-api#55]
The question is not related to library, I need a help so thought to ask it here.
 it's regarding parsing date of listing publication date, so the date is in format "23/09/2021 10:34:38 MEST" i want to convert it to UTC since different marketplace can have different timezone it's hard to parse it in JS with even libraries without having proper timezone name. Thanks for your comment in advance.

**Comment by amz-tools** [comment]
Hi @iamsr,

try using [this npm package](https://www.npmjs.com/package/@vvo/tzdb). It has a function `getTimeZones()`, which will return an array of timezones with all the info you need, including the abbreviation of the timezone. You can look up the timezone name and should be able to use that with moment.js `tz()` function to format it and get the right UTC.

**Comment by iamsr** [comment]
ok thanks @amz-tools
